src/components/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -370,7 +439,44 @@ export default {
 		noUpdate() {
 			this.updateToCurrentPaymentMethod = true;
 		},
-	},
+		onConfirmCancel() {
+			this.$emit('cancel-subscription');
+			this.showCancelLightbox = false;
+			this.modalStep = '';
+		},
+		onModifyCancel() {
+			this.modalStep = CHANGE_SUBSCRIPTION;
+			this.showEditLightbox = true;
+			this.showCancelLightbox = false;
+		},
+		onAbortCancel() {
+			this.showCancelLightbox = false;
+		},
+		trackStep(step) {
+			switch (step) {
+				case CHANGE_SUBSCRIPTION:
+					this.$kvTrackEvent('monthlyGood', 'click-edit-monthly-good', 'Edit');
```

Comment:
These 3 events do not conform to our new web events framework. 

The category should be "subscription", the action "click", and the label something descriptive